spec/factories/targets.rb

```diff
@@ -6,5 +6,6 @@
     target_count 100
     unit Target.units[:submissions]
     conference
+    campaign
   end
```

Comment:
For target factory we should keep only necessary attributes/relationships.
However, you can introduce a new factory, eg target_full which will have more relationships, and you can work on that. Check https://github.com/openSUSE/osem/blob/master/spec/factories/conferences.rb